src/uwtools/drivers/orog.py

```diff
@@ -0,0 +1,125 @@
+"""
+A driver for UFS_UTILS's orog.
+"""
+
+from pathlib import Path
+
+from iotaa import asset, external, task, tasks
+
+from uwtools.drivers.driver import DriverTimeInvariant
+from uwtools.drivers.support import set_driver_docstring
+from uwtools.strings import STR
+from uwtools.utils.file import writable
+from uwtools.utils.tasks import filecopy, symlink
+
+
+class Orog(DriverTimeInvariant):
+    """
+    A driver for orog.
+    """
+
+    # Workflow tasks
+
+    @tasks
+    def files_copied(self):
+        """
+        Files copied for run.
+        """
+        yield self.taskname("files copied")
+        yield [
+            filecopy(src=Path(src), dst=self.rundir / dst)
+            for dst, src in self.config.get("files_to_copy", {}).items()
+        ]
+
+    @tasks
+    def files_linked(self):
+        """
+        Files linked for run.
+        """
+        yield self.taskname("files linked")
+        yield [
+            symlink(target=Path(target), linkname=self.rundir / linkname)
+            for linkname, target in self.config.get("files_to_link", {}).items()
+        ]
+
+    @external
+    def grid_file(self):
+        """
+        The input grid file.
+        """
+        grid_file = Path(self.config["grid_file"])
+        yield self.taskname("Input grid file")
+        yield asset(grid_file, grid_file.is_file) if str(grid_file) != "none" else None
+
+    @task
+    def input_config_file(self):
+        """
+        The input config file.
+        """
+        path = self._input_config_path
+        yield self.taskname(str(path))
+        yield asset(path, path.is_file)
+        yield self.grid_file()
+        inputs = self.config.get("old_line1_items")
+        if inputs:
+            ordered_entries = [
+                "mtnres",
+                "lonb",
+                "latb",
+                "jcap",
+                "nr",
+                "nf1",
+                "nf2",
+                "efac",
+                "blat",
+            ]
+            inputs = " ".join([str(inputs[i]) for i in ordered_entries])
+        outgrid = self.config["grid_file"]
+        orogfile = self.config.get("orog_file")
+        mask_only = self.config.get("mask", ".false.")
+        merge_file = self.config.get("merge", "none")  # string none is intentional
+        content = [i for i in [inputs, outgrid, orogfile, mask_only, merge_file] if i is not None]
```

Comment:
Is this for a non-standard / pre-release version of the code? I'm looking [here](https://github.com/ufs-community/UFS_UTILS/blob/develop/sorc/orog_mask_tools.fd/orog.fd/mtnlm7_oclsm.F#L82-L84) and only see three items being read from this config file, which looks like it needs to be called `fort.5`. I might be confused. I'm kind of curious to see how the absence of that apparently-optional line of integers is handled. Do you have a pointer to the code?

Reply:
The older version that SRW is currently stuck on [reads 5 lines here.](https://github.com/ufs-community/UFS_UTILS/blob/57bd8321764f39a9e1926d104a9207f800bcc80d/sorc/orog_mask_tools.fd/orog.fd/mtnlm7_oclsm.F#L86-L90)

```
      READ(5,*) MTNRES,IM,JM,NM,NR,NF0,NF1,EFAC,BLAT
      READ(5,*) OUTGRID
      READ(5,*) INPUTOROG
      READ(5,*) mask_only
      READ(5,*) merge_file
```

Reply:
I was using INPS to stay consistent with the [ush example script from UFS_UTILS](https://github.com/ufs-community/UFS_UTILS/blob/cf2da7672222677f9760f8229c27d6a2450c7080/ush/fv3gfs_make_orog.sh#L106). I am open to other names since it's arbitrary.

Reply:
How about `orog.cfg`, unless the file is actually in some known format such that we could use a `.yaml`, `.ini`, etc. extension. This looks to me like a good chance to replace an inscrutable filename with something that actually means something.

Reply:
It is definitely just a text file. I don't hate `orog.cfg`. Makes sense to me.